src/python/py/models/builder.py

```diff
@@ -45,6 +45,8 @@ def __init__(self, config, io_dtype, onnx_dtype, ep, cache_dir, extra_options):
         self.filename = extra_options["filename"] if "filename" in extra_options else "model.onnx"
         self.hf_token = parse_hf_token(extra_options.get("hf_token", "true"))
         self.extra_options = extra_options
+        self.quant_provider = extra_options["quant_provider"] if "quant_provider" in extra_options else "default"
```

Comment:
Can we add `quant_provider` and `calibration_method` to `self.quant_attrs["int4"]` since these are INT4-specific settings?

https://github.com/microsoft/onnxruntime-genai/blob/7998f13e25f31caaba803db4197f737df99a0bb9/src/python/py/models/builder.py#L277-L283

Reply:
Done